src/include/buffer.h

```diff
@@ -1286,6 +1286,23 @@ inline bufferhash& operator<<(bufferhash& l, const bufferlist &r) {
   return l;
 }
 
+static inline
+void copy_bufferlist_to_iovec(const struct iovec *iov, unsigned iovcnt,
```

Comment:
In C++, in most cases, `inline` is sufficient in header files and `static inline` can generate spurious warnings.

Reply:
OK, I'll remove static